image.yaml

```diff
@@ -32,6 +32,7 @@ envs:
 ports:
 - value: 8080
 - value: 8443
+- value: 8778
```

Comment:
This definitely is in need of a comment. FWIW, I'm not sure why this cannot be enabled by images which add jolokia themselves. Isn't the idea for them to use `FROM openjdk/openjdk-11` for example and then add it back in? Pretty sure this can be achieved with a docker build strategy on OpenShift with subsequent s2i customizations.

Reply:
I think it is possible with a docker build strategy. I didn't think it was possible with a "pure" s2i build (the kind which we exercise when we run the s2i binary outside of openshift) but I might be out-of-date wrt with what it can do. I guess I'm happy to close the port so long as we can document (in the developer's blog post perhaps) how to open it again. I'll take a look at that.

Reply:
Thanks!

Reply:
the current draft blog post points at this (WIP) solution for re-activating Jolokia. It's a pure S2I-based approach at the moment, requiring a custom assemble script and a custom environment https://github.com/jmtd/quarkus-quickstarts/commit/7f9e866aa0dab735ec8d7397f78b40cad46745b5
I don't *think* its possible with just the above to open an additional port,  so I'll double-check that and see how much more complicated it would be to do this another way (Docker build strategy in OpenShift for instance. I have an abandoned blog post about something else that touched on that I might be able to re-use)

Reply:
Understood that s2i might not work, but that's expected (to me). After all the use case would not to build application source with it, but to extend our image with new libraries. A custom docker build strategy for this would make more sense. Users would then use the resulting s2i image to build their apps with jolokia.